Styles.css

```diff
@@ -0,0 +1,318 @@
+.headbuttons {
+  background: black;
+  font-size: 105%;
+  height: 43px;
+  margin: 0px;
+  margin-left: 0px;
+  color: #fff;
+  border-style: none;
+  border-radius: 0px;
+}
+.headbuttons:hover {
+  background: darkgray;
+  color: black;
+}
+button {
```

Comment:
Do not use css like this. If I will put new `button` it will get this styling.

Reply:
All buttons must be in this style

Reply:
all buttons i this page must have this styling